app/Email.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace emailApp;
+
+class Email {
+	function validateEmail($email) {
+
+		$regex = '/^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$/';
+
+		if (!preg_match($regex, $email)) {
+			throw new \InvalidArgumentException("PLEASE ENTER VALID EMAIL ADDRESS.....");
```

Comment:
Never use all caps in error message. All caps is considered as yelling. http://www.newrepublic.com/article/117390/netiquette-capitalization-how-caps-became-code-yelling We should write messages in formal English.  `....` isn't a formal punctuation.